### PR TITLE
Little details

### PR DESCRIPTION
**Pull request by J053Fabi0**
Can you check my fork?
I think you would like it.

### CODE REVIEW

**Comment by J053Fabi0** [comment]
...

File: index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		}
         body {
             font-family: Arial;
-            background-color: #AD8E58;
+            background-color: #5C6BC0;
```
Comment (BoatFloater) [inline index.html]:
```suggestion
            background-color: #6E9E6D;
```

Comment (fuopy) [reply index.html]: Wow, love the design!

Comment (J053Fabi0) [reply index.html]: I made this pull request when I was 15 🤣. I'm glad it finally got accepted.

**Pull request merged**